helusers/tests/test_oidc_api_token_authentication.py

```diff
@@ -4,28 +4,46 @@
 
 from helusers.oidc import ApiTokenAuthentication
 
-from .conftest import encoded_jwt_factory, ISSUER1
+from .conftest import ISSUER1, encoded_jwt_factory
 
 
 @pytest.fixture(autouse=True)
 def auto_auth_server(auth_server):
     return auth_server
 
 
-@pytest.mark.django_db
-def test_valid_jwt_is_accepted(rf, unix_timestamp_now):
-    sut = ApiTokenAuthentication()
+@pytest.fixture
+def user_uuid():
+    return uuid.UUID("b7a35517-eb1f-46c9-88bf-3206fb659c3c")
 
-    user_uuid = uuid.UUID("b7a35517-eb1f-46c9-88bf-3206fb659c3c")
 
-    encoded_jwt = encoded_jwt_factory(
+@pytest.fixture
+def jwt_data(user_uuid, unix_timestamp_now):
+    return dict(
         iss=ISSUER1,
         aud="test_audience",
         iat=unix_timestamp_now - 10,
         exp=unix_timestamp_now + 1000,
         sub=str(user_uuid),
     )
 
+
+@pytest.mark.parametrize(
+    "jwt_data_extra",
+    [
+        dict(),
+        dict(amr="something"),
+        dict(amr=["something"]),
+    ],
+)
+@pytest.mark.django_db
+def test_valid_jwt_is_accepted(rf, jwt_data, user_uuid, jwt_data_extra):
+    sut = ApiTokenAuthentication()
+
+    user_uuid = uuid.UUID("b7a35517-eb1f-46c9-88bf-3206fb659c3c")
```

Comment:
Seems unnecessary to override `user_uuid` with the same value.

Reply:
Oops... forgot to remove this 😅